src/main.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(try_blocks)]
+#![allow(clippy::result_large_err)]
```

Comment:
Out of curiosity what’s clippy saying that makes this necessary?

Reply:
```
warning: the `Err`-variant returned from this function is very large
   --> src/main.rs:221:23
    |
221 |     fn sync(&self) -> Result<(), ureq::Error> {
    |                       ^^^^^^^^^^^^^^^^^^^^^^^ the `Err`-variant is at least 224 bytes
    |
    = help: try reducing the size of `ureq::Error`, for example by boxing large elements or replacing it with `Box<ureq::Error>`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#result_large_err
```